Update wiki tournament `GBC 2024 Spring` (#11070)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2024_Spring/en.md

The rules are largely the same as in the regular stage.

1. The organiser will invite the highest ranking players from the regular stage to participate in the playoffs.

>    - For personal reason, invited players could refuse to play in playoff, the slot will pass on to the players in next rank.

Reviewer: ```suggestion
   - If invited players decline for any reason, their slot will pass on to the players in next rank.
```

| Finals & Grand Finals | 7 RC, 3 HB, 4 LN, and 1 TB |

3. Players will have 2 minutes to protect, pick and ban a beatmap, and 2 minutes to get ready. If a player takes more time than allowed, the procedures adopted will be as follows:
   - For map bans or protects, the player forfeits the ban or protect.
   - For map picks, a random map will be chosen from the mappool using `!roll X`, where `X` is the number of maps that were neither picked nor banned, excluding the tiebreaker.
   - For a ready timer, the referee will use the `!mp start 10` command.
4. Each player will receive one *tactical timeout* of **2 minutes**, to be used as extra time to pick or ban a beatmap. The tactical timeout is optional.
   - The tactical timeout may be called on a tiebreaker, provided that all players agree to it.
5. A referee will create a multiplayer room 10 minutes before the scheduled match time. Players must join the lobby in that period.
   - The room settings are `Game mode: osu!mania`, `Team mode: Team Vs` and `Score mode: ScoreV2`. The room name must follow the pattern of `GBC 2024 Spr: ({Player A}) vs ({Player B})`.
6. Each player must use `!roll` once in `#multiplayer`.

>    - The winner of the `!roll` chooses whether to protect first or ban first. Protect first will also pick first.

Reviewer: ```suggestion
   - The winner of the `!roll` chooses whether to protect first or ban first. The player who protects first will also pick first.
```
